client/src/components/DataImportExport.vue

```diff
@@ -56,11 +56,10 @@ export default defineComponent({
           await loadProject(currentProject.value);
         }
       } catch (ex) {
+        const text = ex || 'Import failed due to server error.';
+        importErrors.value = true;
+        importErrorText.value = text;
         importing.value = false;
-        this.$snackbar({
-          text: ex || 'Import failed. Refer to server logs for details.',
-        });
-        console.error(ex);
```

Comment:
I'd keep this around, seems handy

Reply:
The same error message (`ex`) would be shown in the dialog instead of the temporary snackbar. Should we display both?